frontend/src/components/footer/Footer.tsx

```diff
@@ -1,82 +1,120 @@
 import React from "react"
-import { FacebookIcon, GitHubIcon, InstagramIcon, SlackIcon } from "../../icons"
+import {
+  FacebookIcon,
+  GitHubIcon,
+  InstagramIcon,
+  SlackIcon,
+  BriefcaseIcon,
+  EnvelopeIcon,
+  MobileIcon,
+  FileIcon,
+} from "../../icons"
 
-export const ContactSection = () => (
+export const FeedbackSection = () => (
   <div className="text-slate-12 flex flex-col text-center font-medium">
     <div>Feil på nettsiden?</div>
     <div>
       Ta kontakt med{" "}
-      <a className="text-red-11" href="mailto:dotkom@online.ntnu.no">
+      <a className="text-red-600" href="mailto:dotkom@online.ntnu.no">
         Dotkom
       </a>
     </div>
   </div>
 )
 
-export const SoMeSection = () => {
-  const icons = [
-    <SlackIcon key="slack" />,
-    <GitHubIcon key="github" />,
-    <InstagramIcon key="instagram" />,
-    <FacebookIcon key="facebook" />,
+interface FooterContactInfoGroup {
+  title: string
+  items: React.ReactNode[]
+}
+
+export const ContactInfoSection = () => {
+  const contactInfoGroups: FooterContactInfoGroup[] = [
+    {
+      title: "Besøksadresse",
+      items: ["A-blokka, A4-137", "Høgskoleringen 5", "NTNU Gløshaugen"],
+    },
+    {
+      title: "Kontaktinformasjon",
+      items: [
+        <p>
+          <BriefcaseIcon key="briefcase" />
+          992 548 045 (OrgNr)
+        </p>,
+        <p>
+          <EnvelopeIcon key="envelope" />
+          <a href="mailto:kontakt@online.ntnu.no">kontakt@online.ntnu.no</a>
+        </p>,
+        <p>
+          <FileIcon key="file" />
+          <a href="mailto:okonomi@online.ntnu.no">okonomi@online.ntnu.no</a>
+        </p>,
+        <p>
+          <MobileIcon key="mobile" />
+          454 61 372
+        </p>,
```

Comment:
I think it would look a little bit nicer if we have some more space between the icons and the text, see

![image](https://github.com/dotkom/vengeful-vineyard/assets/42585241/ab9a097d-b76c-4aa3-8d13-220e6df865e8)

I believe the easiest way to do this, is to make the `<p>` elements `inline-flex gap-1` or some other gap number

Reply:
Something like this? 
![image](https://github.com/dotkom/vengeful-vineyard/assets/78028923/47c6171f-805d-43e2-948d-d7fa8441bfd7)

Reply:
That looks great